src/modules/transaction/utils.ts

```diff
@@ -0,0 +1,27 @@
+import { race, take } from 'redux-saga/effects'
+import {
+  FetchTransactionSuccessAction,
+  FetchTransactionFailureAction,
+  FETCH_TRANSACTION_SUCCESS,
+  FETCH_TRANSACTION_FAILURE
+} from 'decentraland-dapps/dist/modules/transaction/actions'
+
+// TODO: Move to a lib like decentraland-dapps
+// Function that blocks the execution and waits until a tx is mined or it fails
+export function* waitForTx(txHash: string) {
+  while (true) {
```

Comment:
we can reuse this on `item/sagas.ts` file right? on `handleRescueItemsRequest`

Reply:
absolutely